docs/current_thinking.rst

```diff
@@ -8,15 +8,15 @@ This document simply outlines some of the current rationale that has an influenc
 
 - Not dealing with external documents.
 
-  - Absolves libcellml from fetching files and communicating across the internet.
+  - Absolves libcellml from fetching files and communicating across the Internet.
   - Expect to provide another layer that would perform this role as a separate thing.
   - No avenue to retrieve external references.
 
 - Serialise and deserialise from a string.
-- Present a useful interface not one tied to XML structure.
-- Validation is going to be quite separate (you are free to make invalid CellML models).
-- Not creating our own MathML object model
+- Present a useful interface not one tied to `XML <https://www.w3.org/XML/>`_ structure.
+- Validation is going to be quite separate (you are free to make invalid `CellML <https://www.cellml.org/>`_ models).
+- Not creating our own `MathML <https://www.w3.org/Math/>`_ object model.
```

Comment:
Is this still true?

Reply:
Which bit? Either way, it would probably be worth having a new issue to update the contents of the documentation itself (as opposed to reformatting it, etc.).